### ISSUE TITLE
Firefox, Vivaldi and Edge not playing with various subtitles

### ISSUE THREAD

**Issue by Evoley** [jellyfin/jellyfin#695]
**Describe the bug**
After #376 was fixed.. I started having problems with various subtitles (ASS, PGSSUB, SUBRIP) on Firefox, Waterfox, Vivaldi and Edge. The browsers would just continuously load the video file.

**To Reproduce**
1. Open any tv show or movie with ASS, PGSSUB or SUBRIP subtitles
2. Wait for it to load
3. Wait even more (I've waited for about 5 minutes.

**Expected behavior**
Video to play with subtitles.

**Logs**
[ffmpeg-remux.Vivaldi.txt](https://github.com/jellyfin/jellyfin/files/2791627/ffmpeg-remux.Vivaldi.txt)
[ffmpeg-transcode.Firefox.3.txt](https://github.com/jellyfin/jellyfin/files/2791628/ffmpeg-transcode.Firefox.3.txt)
[ffmpeg-transcode-ca132aa4-8a67-4e5b-9e50-ddb404b1db77.txt](https://github.com/jellyfin/jellyfin/files/2791629/ffmpeg-transcode-ca132aa4-8a67-4e5b-9e50-ddb404b1db77.txt)
[ffmpeg-transcode-cfdecfcc-c105-400e-a730-3a1bab300b3b.txt](https://github.com/jellyfin/jellyfin/files/2791630/ffmpeg-transcode-cfdecfcc-c105-400e-a730-3a1bab300b3b.txt)
[ffmpeg-transcode-Firefox.2.txt](https://github.com/jellyfin/jellyfin/files/2791631/ffmpeg-transcode-Firefox.2.txt)
[ffmpeg-transcode-Firefox.txt](https://github.com/jellyfin/jellyfin/files/2791632/ffmpeg-transcode-Firefox.txt)
[log_20190123.Firefox.log](https://github.com/jellyfin/jellyfin/files/2791633/log_20190123.Firefox.log)
[log_20190123.log](https://github.com/jellyfin/jellyfin/files/2791634/log_20190123.log)
[log_20190123.Vivaldi.log](https://github.com/jellyfin/jellyfin/files/2791635/log_20190123.Vivaldi.log)
[log_20190124.log](https://github.com/jellyfin/jellyfin/files/2791636/log_20190124.log)
[ffmpeg-remux-35a2b46c-6b47-4375-8c04-11699859bf16.txt](https://github.com/jellyfin/jellyfin/files/2791637/ffmpeg-remux-35a2b46c-6b47-4375-8c04-11699859bf16.txt)

**System (please complete the following information):**
 - OS: Windows 10 x64
 - Browser: Firefox, Vivaldi, Waterfox, Microsoft Edge
 - Jellyfin Version: 10.1.0

**Additional context**
Chrome is the only browser that works perfectly fine. DVDSUB works fine too.

**Comment by cvium** [comment]
Just fyi, the Hi10 error is another issue (please create one for that as well).

**Comment by cvium** [comment]
I can't reproduce the issue with external and embedded SRT in Firefox on macOS

**Comment by Evoley** [comment]
It might of been due to the videos being hi10. I'll close this for now.
